### PR TITLE
Feature ss/picard collect multiple metric

### PR DESCRIPTION
**Pull request by ionox0**

### CODE REVIEW

File: picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl [hunk 1]
```diff
@@ -0,0 +1,215 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: picard_mark_duplicates_2_8_1
```
Comment (rhshah) [inline picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: should be collect_multiple_meties

File: picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+      separate: false
+    doc: >-
+      Include unpaired reads in CollectSequencingArtifactMetrics. If set to true
+      then all paired reads will be included as well - MINIMUM_INSERT_SIZE and
+      MAXIMUM_INSERT_SIZE will be ignored in CollectSequencingArtifactMetrics.
+      Default value: false. This option can be set to 'null' to clear the
+      default value. Possible values: {true, false}
+outputs:
+  - id: all_metrics
+    type: Directory
```
Comment (rhshah) [inline picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: Why is this a directory, shouldn't we get all the files related to a basename?

File: picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+      ${
+          if(inputs.output_file_name){
+              return inputs.output_file_name
+          } else {
+              return inputs.input.basename.replace(/.bam/,'_md.bam')
+          }
+      }
+requirements:
+  - class: ResourceRequirement
+    ramMin: "${\r  if(inputs.memory_per_job && inputs.memory_overhead) {\r   \r    return inputs.memory_per_job + inputs.memory_overhead\r  }\r  else if (inputs.memory_per_job && !inputs.memory_overhead){\r    \r   \treturn inputs.memory_per_job + 2000\r  }\r  else if(!inputs.memory_per_job && inputs.memory_overhead){\r    \r    return 15000 + inputs.memory_overhead\r  }\r  else {\r    \r  \treturn 17000 \r  }\r}"
```
Comment (rhshah) [inline picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: Need to comment ramMin and coreMin that have expression and add static values to it

File: picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+    'foaf:member':
+      - class: 'foaf:Person'
+        'foaf:mbox': 'mailto:shahr2@mskcc.org'
+        'foaf:name': Ronak Shah
+    'foaf:name': Memorial Sloan Kettering Cancer Center
+'dct:creator':
+  - class: 'foaf:Organization'
+    'foaf:member':
+      - class: 'foaf:Person'
+        'foaf:mbox': 'mailto:shahr2@mskcc.org'
```
Comment (rhshah) [inline picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: Need to add your names here :)

**Comment by rhshah** [comment]
@ionox0  @shalabhsuman  Can you also modify the /docs/SUMMARY.md file to point to the README.md for this tools

**Comment by rhshah** [comment]
#4

File: picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl (hunk 2 above)
Comment (shalabhsuman) [reply picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: @ionox0 and I discussed this and decided to change it from basename to Directory. @ionox0 Do you want to comment on this?

Comment (rhshah) [reply picard_collectmultiplemetric_2.8.1/picard_collectmultiplemetrics_2-8-1.cwl]: https://gitlab.ddbj.nig.ac.jp/genomics-workflow-standards/human-reseq/blob/734dc99af9d2ea36d1891753847408576449a74d/Tools/picard-CollectMultipleMetrics.cwl

**Pull request merged**